make prep-dom0 fails when invoked directly

This error does not occur when running the targets are run during a `make all` run, only when invoked directly through the `prep-dom0` Makefile target.
#### Steps to reproduce:
1. checkout latest master
2. `make prep-dom0`
3. observe error:
```
Deploying Salt config...
local:
    ----------
    sd-workstation.top:
        ----------
        status:
            unchanged
sudo qubesctl top.enable sd-vm-updates
[ERROR   ] An un-handled exception was caught by salt's global exception handler:
SaltRenderError: Could not find relpath for sd-vm-updates.top
Traceback (most recent call last):
  File "/bin/qubesctl", line 98, in <module>
    salt_call()
  File "/usr/lib/python2.7/site-packages/salt/scripts.py", line 391, in salt_call
    client.run()
  File "/usr/lib/python2.7/site-packages/salt/cli/call.py", line 58, in run
    caller.run()
  File "/usr/lib/python2.7/site-packages/salt/cli/caller.py", line 134, in run
    ret = self.call()
  File "/usr/lib/python2.7/site-packages/salt/cli/caller.py", line 204, in call
    ret['return'] = func(*args, **kwargs)
  File "/var/cache/salt/minion/extmods/modules/topd.py", line 74, in enable
    return TopUtils(__opts__, **kwargs).enable(paths, saltenv)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 600, in enable
    toppaths, unseen = self.prepare_paths(paths)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 453, in prepare_paths
    toppath = self.toppath(path)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 332, in toppath
    saltenv = saltenv or self.saltenv(path, saltenv)
  File "/var/cache/salt/minion/extmods/utils/pathutils.py", line 128, in saltenv
    relpath = self.relpath(path)
  File "/var/cache/salt/minion/extmods/utils/pathutils.py", line 486, in relpath
    return self.path(path, saltenv)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 269, in path
    return super(TopUtils, self).path(path, saltenv, path_type=path_type)
  File "/var/cache/salt/minion/extmods/utils/pathutils.py", line 445, in path
    raise SaltRenderError('Could not find relpath for {0}'.format(path))
SaltRenderError: Could not find relpath for sd-vm-updates.top
Traceback (most recent call last):
  File "/bin/qubesctl", line 98, in <module>
    salt_call()
  File "/usr/lib/python2.7/site-packages/salt/scripts.py", line 391, in salt_call
    client.run()
  File "/usr/lib/python2.7/site-packages/salt/cli/call.py", line 58, in run
    caller.run()
  File "/usr/lib/python2.7/site-packages/salt/cli/caller.py", line 134, in run
    ret = self.call()
  File "/usr/lib/python2.7/site-packages/salt/cli/caller.py", line 204, in call
    ret['return'] = func(*args, **kwargs)
  File "/var/cache/salt/minion/extmods/modules/topd.py", line 74, in enable
    return TopUtils(__opts__, **kwargs).enable(paths, saltenv)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 600, in enable
    toppaths, unseen = self.prepare_paths(paths)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 453, in prepare_paths
    toppath = self.toppath(path)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 332, in toppath
    saltenv = saltenv or self.saltenv(path, saltenv)
  File "/var/cache/salt/minion/extmods/utils/pathutils.py", line 128, in saltenv
    relpath = self.relpath(path)
  File "/var/cache/salt/minion/extmods/utils/pathutils.py", line 486, in relpath
    return self.path(path, saltenv)
  File "/var/cache/salt/minion/extmods/utils/toputils.py", line 269, in path
    return super(TopUtils, self).path(path, saltenv, path_type=path_type)
  File "/var/cache/salt/minion/extmods/utils/pathutils.py", line 445, in path
    raise SaltRenderError('Could not find relpath for {0}'.format(path))
salt.exceptions.SaltRenderError: Could not find relpath for sd-vm-updates.top
DOM0 configuration failed, not continuing
Makefile:126: recipe for target 'prep-dom0' failed
make: *** [prep-dom0] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -122,9 +122,7 @@ flake8: ## Lints all Python files with flake8
 template: ## Builds securedrop-workstation Qube template RPM
 	./builder/build-workstation-template
 
-prep-dom0: prep-salt # Copies dom0 config files for VM updates
-	sudo qubesctl top.enable sd-vm-updates
-	sudo qubesctl top.enable sd-dom0-files
+prep-dom0: prep-salt # Copies dom0 config files
 	sudo qubesctl --show-output --targets dom0 state.highstate
 
 destroy-all: ## Destroys all VMs managed by Workstation salt config

